sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ProtoDescriptorRenderer.scala

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.springsdk.impl
+
+import com.google.protobuf.Descriptors.FileDescriptor
+
+import scala.collection.mutable
+
+object ProtoDescriptorRenderer {
+
+  def toString(fileDescriptor: FileDescriptor): String = {
+    // not water tight but better than the default non-protobuf Proto-toString format
+    val proto = fileDescriptor.toProto
+    val builder = new mutable.StringBuilder()
+    proto.getMessageTypeList.forEach { messageType =>
+      builder ++= "message "
```

Comment:
Or maybe better to start the render with a break line? 
```suggestion
      builder ++= "\n"
      builder ++= "message "
```